[2024-05-07T08:09:04.435+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-07T08:09:04.497+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: trigger_airbyte_job_example.airbyte_check_sync manual__2024-05-07T08:08:48.989979+00:00 [queued]>
[2024-05-07T08:09:04.564+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: trigger_airbyte_job_example.airbyte_check_sync manual__2024-05-07T08:08:48.989979+00:00 [queued]>
[2024-05-07T08:09:04.565+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-07T08:09:04.649+0000] {taskinstance.py:2327} INFO - Executing <Task(AirbyteJobSensor): airbyte_check_sync> on 2024-05-07 08:08:48.989979+00:00
[2024-05-07T08:09:04.680+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=204) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-07T08:09:04.698+0000] {standard_task_runner.py:63} INFO - Started process 206 to run task
[2024-05-07T08:09:04.683+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'trigger_airbyte_job_example', 'airbyte_check_sync', 'manual__2024-05-07T08:08:48.989979+00:00', '--job-id', '159', '--raw', '--subdir', 'DAGS_FOLDER/dag_aribyte_ex.py', '--cfg-path', '/tmp/tmpi1u4s327']
[2024-05-07T08:09:04.707+0000] {standard_task_runner.py:91} INFO - Job 159: Subtask airbyte_check_sync
[2024-05-07T08:09:04.902+0000] {task_command.py:426} INFO - Running <TaskInstance: trigger_airbyte_job_example.airbyte_check_sync manual__2024-05-07T08:08:48.989979+00:00 [running]> on host c811ecd915dc
[2024-05-07T08:09:05.282+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='trigger_airbyte_job_example' AIRFLOW_CTX_TASK_ID='airbyte_check_sync' AIRFLOW_CTX_EXECUTION_DATE='2024-05-07T08:08:48.989979+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-07T08:08:48.989979+00:00'
[2024-05-07T08:09:05.285+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-07T08:09:05.298+0000] {baseoperator.py:399} WARNING - AirbyteJobSensor.execute cannot be called outside TaskInstance!
[2024-05-07T08:09:05.340+0000] {base.py:84} INFO - Using connection ID '***_call_airbyte' for task execution.
[2024-05-07T08:09:05.535+0000] {airbyte.py:104} INFO - Waiting for job 261 to complete.
[2024-05-07T08:10:05.543+0000] {base.py:84} INFO - Using connection ID '***_call_airbyte' for task execution.
[2024-05-07T08:10:05.575+0000] {airbyte.py:99} INFO - Job 261 completed successfully.
[2024-05-07T08:10:05.576+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-05-07T08:10:05.576+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-07T08:10:05.585+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=trigger_airbyte_job_example, task_id=airbyte_check_sync, execution_date=20240507T080848, start_date=20240507T080904, end_date=20240507T081005
[2024-05-07T08:10:05.638+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-07T08:10:05.670+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-07T08:10:05.676+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
